app/assets/stylesheets/articles.scss

```diff
@@ -523,44 +265,36 @@
         margin-right: 4px;
       }
       h4 {
-        font-size: 14px;
-        margin-left: 0px;
-        max-width: 80%;
         padding: 0px;
-        margin: 5px 0px;
+        margin: 0;
         font-weight: 500;
-        display: inline-block;
-        left: 62px;
         position: absolute;
-        bottom: 80px;
+        bottom: 50px;
+        left: 72px;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        font-size: 0.8em;
         a {
           @include themeable(color, theme-secondary-color, #666666);
         }
-        @media screen and (min-width: 430px) {
-          bottom: 90px;
-          left: 72px;
-          font-size: 17px;
-        }
+
         &:hover {
           opacity: 0.9;
           text-decoration: underline;
         }
       }
       .tags {
         margin-top: 0px;
-        font-size: 14px;
+        font-size: 0.8em;
```

Comment:
I'm assuming you want this `font-size` relative to the parent? If not, use `rem`.